lib/capistrano/tasks/yarn.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+desc 'Install Javascript dependencies via `yarn`'
+task :yarn_install do
+  on roles(:web) do
+    within release_path do
+      execute("cd #{release_path} && yarn install")
```

Comment:
Do you know why this is needed?  H2 doesn't use this and it has yarn packages.

Reply:
This is handled by `capistrano/rails` and there is no need for a separate capistrano task locally.  See H2:

```
00:41 deploy:assets:precompile
      01 /usr/local/rvm/bin/rvm default do bundle exec rake assets:precompile
      01 yarn install v1.22.15
```